### PR TITLE
Filter panel in separate window with toolbar at top

### PR DESCRIPTION
**Pull request by jdenisgiguere**
Querier has its own window. We use the new toolbarType property to place the toolbar at the top.

Proposal for https://github.com/georchestra/georchestra/issues/1012

### CODE REVIEW

**Comment by jdenisgiguere** [comment]
https://github.com/jdenisgiguere/georchestra/blob/FilterPanel_toolbar/mapfishapp/src/main/webapp/app/js/GEOR.js#L425-L459

May not be usuful anymore (except 1 brace and comma).

File: mapfishapp/src/main/webapp/app/js/GEOR.js [hunk 1]
```diff
@@ -400,7 +400,29 @@ Ext.namespace("GEOR");
             var querierTitle;
             GEOR.querier.events.on({
                 "ready": function(panelCfg) {
+
+                    win = new Ext.Window({
```
Comment (fvanderbiest) [inline mapfishapp/src/main/webapp/app/js/GEOR.js]: Codacy Issue found: [Variable 'win' implicitly declared.](https://www.codacy.com/app/francois-vanderbiest/georchestra/file/2313774630/issues/source?bid=3200181&fileBranchId=3200201#l404)

**Comment by fvanderbiest** [comment]
This PR does allow queries on several layers at once. This is problematic, since we have a singleton here (`GEOR.querier`), whose state (internal vars) are modified by the first query.

We have two options here:
 * either change GEOR.querier into a fully fledged object, thus allowing several instances of it
 * or prevent queries to several layers at once.

We official sold solution 2 to Rennes, but if solution 1 is easier (and I think it is, besides being nicer for the end user), why not giving it a try ? I'll have a look at it this morning.

**Comment by fvanderbiest** [comment]
Work is still in progress, and I'm finally nearly happy with it.
capture du 2016-04-01 11 17 23

**Comment by fvanderbiest** [comment]
Replace by https://github.com/georchestra/georchestra/pull/1346

**Pull request closed without merging**